src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -228,6 +228,8 @@
 
       <Crossgen2SupportedRids Include="@(Net60Crossgen2SupportedRids)" />
 
+      <ILCompilerSupportedRids Include="linux-musl-x64;linux-x64;win-x64;linux-arm;win-arm64;linux-arm64" />
```

Comment:
`linux-arm` is actually arm32 and is not supported currently AFAIK

Reply:
removed linux-arm.